packages/core/src/mpi/normalize-patient.ts

```diff
@@ -28,6 +28,46 @@ export function normalizePatient<T extends PatientData>(patient: T): T {
     lastName,
     contact: (patient.contact ?? []).map(contact => ({
       ...contact,
+      // TODO WE PUT EMAIL IN PHONE SLOT SO THEN NORMALIZE PHONE KILLS EMAIL
+      email: contact.email ? normalizeEmail(contact.email) : contact.email,
+      phone: contact.phone ? normalizePhoneNumber(contact.phone) : contact.phone,
+    })),
+    address: (patient.address ?? []).map(addr => {
+      try {
+        const newAddress: Address = {
+          addressLine1: normalizeAddress(addr.addressLine1),
+          city: normalizeString(addr.city),
+          zip: normalizeZipCode(addr.zip),
+          state: addr.state,
+          country: addr.country || "USA",
+        };
+        if (addr.addressLine2) {
+          newAddress.addressLine2 = normalizeAddress(addr.addressLine2);
+        }
+        return newAddress;
+      } catch (err) {
+        const msg = `Failed to parse the address for MPI`;
+        log(`${msg} - error ${errorToString(err)}`);
+      }
+      return;
+    }),
+  };
+  return normalizedPatient;
+}
+
+export function normalizePatientInboundMpi<T extends PatientData>(patient: T): T {
```

Comment:
I created a new normalizationFunction that doesnt do what the current one does:

```
const [firstName = patient.firstName] = splitName(normalizeString(patient.firstName));
  const [lastName = patient.lastName] = splitName(normalizeString(patient.lastName));
```
this code in normalizePatient splits the strings on the commas and then only uses the first one. Its used in a bunch of places so I didnt want to change it and break a bunch of other things